src/Microsoft.Performance.SDK.Runtime/Extensibility/DataExtensions/Tables/ITableExtensionReference.cs

```diff
@@ -15,5 +15,9 @@ public interface ITableExtensionReference
           IDataExtensionDependency,
           IDisposable
     {
+        /// <summary>
+        ///     Gets a value indicating whether this table is an internal table.
```

Comment:
Have we defined what internal table means anywhere in the docs?

Reply:
I guess it depends on what docs you're asking about.

On TableAttribute:
        /// <param name="internalTable">
        ///     When this is set, the table will not be exposed through the data extension repository.
        ///     Note that the <see cref="IProcessingSource"/> that owns this will be required to create this table
        ///     when this is set.
        /// </param>